spec/models/statistics/total_savings_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 describe Statistics::TotalSavings do
   describe '#to_s' do
-    context 'there are completed auctions' do
+    context 'there are complete and successful auctions' do
```

Comment:
Oh sorry. Yes, we need to clean up the factories in `auctions.rb` since they are sometimes a bit redundant and unclear